src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContext);
+
+  const mother = getPersonByName(motherName, people);
```

Comment:
Performance Issue: The `getPersonByName` function is called during each render of the `PersonItem` component. This could lead to performance issues if the list of people is large. Consider storing the parents in the `Person` object to avoid unnecessary computations.